tests/spec/s-function.js

```diff
@@ -1,7 +1,177 @@
 /* global describe, it, expect, beforeEach */
+var hasStrictMode = (function () {
+  'use strict';
+
+   return !this;
+}());
+
+var global = (function () {
+   return this;
```

Comment:
I used it in the es6-shim for a long time - it can cause problems in Chrome OS plugins, because they disallow CSP, but that's not a concern for tests.